File: P6-NascarSeasonAPI/NascarPage/NascarPage/Entitys/Galeria.cs
using System.ComponentModel.DataAnnotations;

namespace NascarPage.Entitys
{
    public class Galeria
    {
        [Required]
        public int Id { get; set; }
        [Required]
        public int Ronda { get; set; }
        public string FotoUno { get; set; } = null!;
        public string FotoDos { get; set; } = null!;
        public string FotoTres { get; set; } = null!;
    }
}

File: P6-NascarAngularPage/src/app/componentes/galeria/galeria.ts
export interface LecturaGaleriaDTO {
    id: number;
    ronda: number;
    fotoUno: string;
    fotoDos: string;
    fotoTres: string;
}

export interface CrearGaleriaDTO {
    ronda: number;
    fotoUno: File | null | undefined;
    fotoDos: File | null | undefined;
    fotoTres: File | null | undefined;
}